### ISSUE TITLE
Xbox Wireless Controller on Bluetooth not fully mapped 

### ISSUE THREAD

**Issue by sascha777** [gabomdq/SDL_GameControllerDB#340]
Hi guys,

I am running
`moonlight-embedded-2.4.11-1buster1`
on my PI 3b and connect to my PC running 
`NVIDIA GeForce GTX 750 Ti, GFE 3.20.3.63 (gs_04_22_28196077, 7.1.422.0)`
It connects my Xbox Wireless Controller via Bluetooth
```
Detected Xbox Wireless Controller (050000005e040000fd02000003090000) on /dev/input/event2 as Xbox One Wireless Controller
Detected Xbox Wireless Controller Consumer Control (050000005e040000fd02000003090000) on /dev/input/event3 as Xbox One Wireless Controller
```
which is listed in the gamecontrollerdb.txt and most recently updated.

Issue:
The return/back button isn't mapped correctly.
The verbose log on moonlight-embedded shows (on that same button) the following two missing mappings BUT not on pressing the button but a lit later:
```
Unmapped button: 309
Unmapped button: 313
Unmapped button: 314
Unmapped button: 316
```
Expected behaviour:
Back/return button is working properly

Any ideas why the button isn't mapped correctly?

Thanks for your help!

**Comment by offalynne** [comment]
This map comes from SDL upstream. See https://github.com/SDL-mirror/SDL/commit/d34fb48c6fc29e6ceb22632841b32c17ed970cd9

**Comment by sascha777** [comment]
Thanks for this hint! 
Updating firmware didn't help but I will try to manually check the controllerdb this evening.

**Comment by sascha777** [comment]
Updating firmware did not help. I also tried different unassigned buttons for the back (all other work fine) in the gamecontrollerdb.txt, but that also didn't work.

I then connected the gamepad via USB and disabled bluetooth.
That brings me a verbose output of
`No mapping available for Microsoft X-Box One S pad (030000005e040000ea02000008040000) on /dev/input/event2`

Is there anything else that I could try?
